File: 151004/Terebey/RestApp/src/main/java/com/distributed_computing/rest/bean/Creator.java
package com.distributed_computing.rest.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Creator {
    int id;
    String login;
    String password;
    String firstname;
    String lastname;
}

File: 151002/Haiduk/RV/RV/Lab5/Lab5.Publisher/Models/Creator.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Lab5.Publisher.Models;

[Table("tbl_Creator")]
public class Creator : BaseModel
{
    [Required]
    [MinLength(2)]
    [MaxLength(64)]
    [Column(TypeName = "text")]
    public string Firstname { get; set; }

    [Required]
    [MinLength(2)]
    [MaxLength(64)]
    [Column(TypeName = "text")]
    public string Lastname { get; set; }

    [Required]
    [MinLength(2)]
    [MaxLength(64)]
    [Column(TypeName = "text")]
    public string Login { get; set; }

    [Required]
    [MinLength(8)]
    [MaxLength(128)]
    [Column(TypeName = "text")]
    public string Password { get; set; }

    public virtual ICollection<News> News { get; set; } = [];
}